test/src/blockMonitor/component.test.ts

```diff
@@ -143,7 +144,7 @@ describe("MappedStateReducer", () => {
         expect(reducedState.items).to.deep.equal(expectedMap);
     });
 
-    it("reduce calls getInitialState if a new object id is added to the collection", () => {
+    fnIt<MappedStateReducer<any,any,any,any>>(m => m.reduce, "calls getInitialState if a new object id is added to the collection", () => {
```

Comment:
Wow, this is interesting, I hadn't thought about generics. I think using the <any> is fine, maybe you should point that out in the documentation you write on `fnIt`